x-pack/legacy/plugins/lens/server/usage/task.ts

```diff
@@ -11,14 +11,13 @@ import { CallClusterOptions } from 'src/legacy/core_plugins/elasticsearch';
 // This import has the side effect of allowing us to use the elasticsearch type
 // extensions below. Without this import, the compiler is unable to find these
 // in tests
-import {} from '../../../apm/typings/elasticsearch';
 import {
```

Comment:
There's a comment here explaining this import, but you haven't moved the comment